cirq-superstaq/cirq_superstaq/compiler_output_test.py

```diff
@@ -222,6 +222,14 @@ def test_read_json_aqt() -> None:
 
     assert out.seq is None
 
+    # if num_eca_circuits
+    out = css.compiler_output.read_json_aqt(json_dict, circuits_is_list=True, num_eca_circuits=2)
+    assert out.circuits == [[circuit, circuit]]
+    assert out.final_logical_to_physicals == [
+        [final_logical_to_physical, final_logical_to_physical]
+    ]
+    assert out.pulse_lists is None
+
```

Comment:
Did you add this? Only asking cause it's not related to the PR title.

Reply:
Yes, removing ` # pylint: disable=missing-param-doc` in `compiler_output.py` failed the coverage test so I added that test.

Reply:
I commented out this code in https://github.com/Infleqtion/client-superstaq/pull/586 and the checks pass.

Reply:
Yeah saw the test PR too, removed the tests with [6d115a3](https://github.com/Infleqtion/client-superstaq/pull/565/commits/6d115a3bf55825703943d785e30f050482397e39) (not sure why I saw lines 231-239 in `cirq_superstaq/compiler_ouput.py` flagged during one of my earlier `./check/all_.py -i` checks locally)